docs/spec/lightclient/detection/detection.md

```diff
@@ -149,7 +337,7 @@ false* then we have a *slashable fork*.
 > full nodes as part of normal Tendermint consensus protocol". Please
 > confirm! 
   
-#### **[TMBC-SIGN-UNIQUE]**
+#### **[TMBC-SIGN-UNIQUE.1]**
```

Comment:
The same comment here: we need to give full condition and not saying sign-unique(b,c) is false as it can be false when heights differ.